templates/zerver/api/get-all-streams.md

```diff
@@ -79,7 +79,7 @@ A typical successful JSON response may look like:
 
 {generate_code_example|get-all-streams|fixture}
 
-An example of a JSON response for when the user is not authorized
-to use the `include_all_active` parameter:
+An example JSON response for when the user is not authorized to use the
+`include_all_active` parameter:
```

Comment:
This would benefit from e.g. "because they are not an organization administrator" or whatever the reason is.